lib/matplotlib/backends/qt_editor/figureoptions.py

```diff
@@ -235,15 +235,25 @@ def apply_callback(data):
 
         # re-generate legend, if checkbox is checked
         if generate_legend:
-            draggable = None
-            ncol = 1
+            kwargs = {}
             if axes.legend_ is not None:
                 old_legend = axes.get_legend()
-                draggable = old_legend._draggable is not None
-                ncol = old_legend._ncol
-            new_legend = axes.legend(ncol=ncol)
-            if new_legend:
-                new_legend.set_draggable(draggable)
```

Comment:
is draggable set lost?

Reply:
Good catch, sorry I missed that.

Reply:
draggable is not an argument to ax.legend()